examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "rvc-mode-delegates.h"
+#include "rvc-operational-state-delegate.h"
+#include <app/clusters/mode-base-server/mode-base-server.h>
+#include <app/clusters/operational-state-server/operational-state-server.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+class RvcDevice
+{
+private:
+    RvcRunMode::RvcRunModeDelegate mRunModeDelegate;
+    ModeBase::Instance mRunModeInstance;
+
+    RvcCleanMode::RvcCleanModeDelegate mCleanModeDelegate;
+    ModeBase::Instance mCleanModeInstance;
+
+    RvcOperationalState::RvcOperationalStateDelegate mOperationalStateDelegate;
+    OperationalState::Instance mOperationalStateInstance;
+
+public:
+    RvcDevice() :
+        mRunModeDelegate(),
+        mRunModeInstance(&mRunModeDelegate, 0x1, RvcRunMode::Id, chip::to_underlying(RvcRunMode::Feature::kOnOff)),
```

Comment:
please replace the integer literal with a constant, here and below

Reply:
See [here](https://github.com/project-chip/connectedhomeip/pull/28719/commits/63c23e20c8f4e9df44048c7839ec41c0d41280f9).